README.md

```diff
@@ -98,7 +98,52 @@ If you don't have a Traefik instance set up, there's a `docker-compose-traefik.y
 
 #### Others (APISIX)
 
-The system is also set up to run using APISIX as the gateway (to an extent). Use the `docker-compose-apisix.yml` file to spin up the application with APISIX and etcd. You will need to define routes for the app and set up forward auth on your own.
+The system is also set up to run using APISIX as the gateway (to an extent). Use the `docker-compose-apisix.yml` file to spin up the application with APISIX and etcd. The APISIX configuration is not ready for production use - it's only here for assist in local testing and development.
+
+You'll need to define routes for APISIX before it will handle traffic for the appplication. Here are the steps to accomplish that:
+
+1. In your Keycloak instance, create a new Client in the realm you are going to use for UE.
+   2. The `Client ID` can be any valid string - a good choice is `apisix-client`.
+   1. For local testing, it's OK to use `*` for both `Valid redirect URIs` and `Web origins`. This is not OK for anything attached to the Internet. 
+   2. Make sure `Client authentication` is on, and `Standard flow` and `Implicit flow` are checked.
+   3. After you've saved the client, go to Credentials and copy out the `Client secret`. (You may need to manually cut and paste; the copy to clipboard button has never worked for me.)
+2. In your Keycloak Realm Settings, you should be able to find the OpenID Endpoint Configuration link. Copy/paste this somewhere - you'll need it later.
+3. From the `config/apisix/apisix.yml` file, get the `key` out. This should be on line 11. You can also reset it here if you wish. 
```

Comment:
Good call; updated the docs appropriately. (Also - I had the realm hard-coded in the route definition so that is now also broken out. As it was you'd have to have a realm named `odltest` for it to work properly.)